ebu_tt_live/bindings/__init__.py

```diff
@@ -598,10 +601,9 @@ def _semantic_push_computed_roles(self, dataset):
         dataset['role_stack'].append(self._computed_roles)
 
     def _semantic_pop_computed_roles(self, dataset):
-        if dataset.get('role_stack') and len(dataset['role_stack']) > 0:
+        if dataset.get('role_stack') and dataset['role_stack']:
             dataset['role_stack'].pop()
-        if dataset.get('metadata_roles') and len(dataset['metadata_roles']) > 0:
-            self._computed_roles.update(dataset['metadata_roles'])
+        if dataset.get('metadata_roles') and dataset['metadata_roles']:
             dataset['metadata_roles'].clear()
```

Comment:
I _think_ we can just do `dataset.get('metadata_roles', set()).clear()` since the `get(key, default)` will always return something.